ietf/doc/expire.py

```diff
@@ -238,3 +227,42 @@ def move_file_to(subdir):
         except Document.DoesNotExist:
             # All uses of this past 2014 seem related to major system failures.
             move_file_to("unknown_ids")
+
+
+def repair_dead_on_expire():
+    by = Person.objects.get(pk=1)  # a.k.a. "(System)"
```

Comment:
We canonically use `Person.objects.get(user__username="(System)")`. Better to stick to that?

Reply:
looks like it's mostly `(name="(System)")` rather than `user__username`?

Reply:
Er, yes, `name`. Sorry, I don't know why I thought I saw that.